### PR TITLE
 R4R: Adjust field name 

### PR DESCRIPTION
**Pull request by zhiqiang-bianjie**

### CODE REVIEW

File: docs/zh/cli-client/gov/submit-proposal.md [hunk 1]
```diff
@@ -27,11 +27,11 @@ iriscli gov submit-proposal --help
 | --software           |           " "                 |  新协议的软件地址                                                                       |       |
 | --switch-height           |       0                     |  新版本协议升级的高度                                                     |       |
 | --threshold        | "0.8"   |  软件升级的阈值                                              |               |
-| token-symbol-at-source   |        | 外部代币名称                                                 | |
+| token-canonical-symbol   |        | 外部代币名称                                                 | |
 | --token-symbol |  | 代币符号 | |
 | --token-name |  | 代币名称 | |
 | --token-decimal |  | 代币最大精度 | |
-| --token-symbol-min-alias |  | 代币最小单温别名 | |
+| --token-min-unit-alias |  | 代币最小单温别名 | |
```
Comment (chengwenxi) [inline docs/zh/cli-client/gov/submit-proposal.md]: 代币最小单位别名

File: client/asset/cli/sendtx.go [hunk 2]
```diff
@@ -255,8 +255,8 @@ func GetCmdEditAsset(cdc *codec.Codec) *cobra.Command {
 
 			tokenId := args[0]
 			name := viper.GetString(FlagName)
-			symbolAtSource := viper.GetString(FlagSymbolAtSource)
-			symbolMinAlias := viper.GetString(FlagSymbolMinAlias)
+			CanonicalSymbol := viper.GetString(FlagCanonicalSymbol)
```
Comment (chengwenxi) [inline client/asset/cli/sendtx.go]: CanonicalSymbol -> canonicalSymbol

File: tests/cli/asset_test.go [hunk 3]
```diff
@@ -35,8 +35,8 @@ func TestIrisCLIToken(t *testing.T) {
 	name := "Bitcoin"
 	initialSupply := 2000000000
 	decimal := 18
-	symbolAtSource := "Btc"
-	symbolMinAlias := "Satoshi"
+	CanonicalSymbol := "Btc"
```
Comment (chengwenxi) [inline tests/cli/asset_test.go]: CanonicalSymbol -> canonicalSymbol

**Comment by zhiqiang-bianjie** [comment]
Resolves : #1791

**Pull request merged**